docs/docs/experimentation-analysis/data-pipeline.mdx

```diff
@@ -9,19 +9,92 @@ slug: /app/data-pipeline
+- **Incremental refresh (Recommended)** — On each update, only scans new rows in your experiment assignment and metric sources, and persists reusable intermediate tables across updates to minimize re-scans and speed up analyses.
+- **Ephemeral (Legacy)** — Per-analysis, short-lived tables are created with limited retention for reuse within an analysis window.
 
-To enable Pipeline Mode, follow the steps for your data warehouse:
+## Incremental refresh (Recommended)
 
-## Configuring Pipeline Mode
+Our main pipeline mode reduces cost and speeds up experiment updates by scanning only new data since the last run, while keeping a small set of reusable, de-duplicated tables in your warehouse.
+
+Incremental refresh is only available for BigQuery. Please reach out to support if your warehouse is not supported.
+
+### How it works
+
+On each experiment update, GrowthBook only scans new data in your warehouse to keep the amount of data consumed in each update to a minimum. On each update, GrowthBook materializes several tables in your warehouse to ensure old data does not need to be scanned again, and to reduce the computation needed in future analyses.
+
+Specifically, for the experiment assignment query and fact tables, GrowthBook uses the max timestamp seen in the previous update and only scans for data since that last update, inserting that data into materialized intermediate tables to be used for analysis.
+
+### Main benefits
+
+- Dramatic reduction in the number of rows scanned per update, as GrowthBook only has to scan your experiment assignment queries and fact tables for data in partitions that are on or after the last timestamp seen in the previous update.
+- Dramatic reduction in the cost and waiting time when running on-demand dimension analyses, as GrowthBook can use the existing materialized `Experiment Units Table` and `Units-Date Metrics Table`s to get dimension breakdowns rather than rescanning data in your warehouse.
+
+### Requirements
+
+- A dedicated schema/dataset with write access for GrowthBook.
+- Ability to retain intermediate tables beyond 24 hours (often days) for reuse.
+- Appropriate permissions to create/read/update (and when supported, merge/partition) tables.
+- Experiment assignment queries and fact tables partitioned on the `timestamp` column. If you have a different partitioning strategy, please reach out to support.
+
+To see specific instructions for your warehouse, please see the [Configuration](#configuration) section below.
+
+### Controls and observability
+
+- Default "Update" performs an incremental refresh
+- "Full refresh" is available when you need to re-state results (e.g., backfills, config changes)
+
+### Limitations and notes
+
+- While in Beta, some analysis tools (activation metrics, quantile metrics, ratio metrics with different numerator and denominator fact tables, dimension breakdowns in Experiment Dashboards) are not supported.
```

Comment:
Is it these exact tools? If so, let's just say "the following analysis tools aren't supported:
- activation metrics
- ...

Doesn't have to be a list, but if we know the exact gamut of unsupported tools, let's say that.